examples/docs_snippets/docs_snippets/guides/dagster/reexecution/setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup  # type: ignore
```

Comment:
because this is inside the docs_snippets folder which already is a package ([here](https://github.com/dagster-io/dagster/blob/master/examples/docs_snippets/setup.py)), we don't need to this setup.py here.

Reply:
Makes sense, removing